pkg/tfgen/generate_schema_test.go

```diff
@@ -41,24 +45,34 @@ import (
 // To update the contents of test_data/regress-611-schema.json run the test with env var PULUMI_ACCEPT set to "true".
 func TestRegress611(t *testing.T) {
 	provider := testprovider.ProviderRegress611()
+	expectedFile := abs(t, "test_data/regress-611-schema.json")
+
 	schema, err := GenerateSchema(provider, diag.DefaultSink(io.Discard, io.Discard, diag.FormatOptions{
 		Color: colors.Never,
 	}))
 	assert.NoError(t, err)
-	bridgetesting.AssertEqualsJSONFile(t, "test_data/regress-611-schema.json", schema)
+	bridgetesting.AssertEqualsJSONFile(t, expectedFile, schema)
 }
 
 func TestRegressMiniRandom(t *testing.T) {
 	provider := testprovider.ProviderMiniRandom()
+	expectedFile := abs(t, "test_data/regress-minirandom-schema.json")
+	enterProviderModule(t,
+		"github.com/terraform-providers/terraform-provider-random v1.0.0")
 	schema, err := GenerateSchema(provider, diag.DefaultSink(io.Discard, io.Discard, diag.FormatOptions{
 		Color: colors.Never,
 	}))
 	assert.NoError(t, err)
-	bridgetesting.AssertEqualsJSONFile(t, "test_data/regress-minirandom-schema.json", schema)
+	bridgetesting.AssertEqualsJSONFile(t, expectedFile, schema)
 }
 
 func TestNestedMaxItemsOne(t *testing.T) {
 	provider := testprovider.ProviderMiniCloudflare()
+
+	expectedFile := abs(t, "test_data/regress-minicloudflare-schema.json")
+	enterProviderModule(t,
+		"github.com/terraform-providers/terraform-provider-cloudflare v1.0.0")
```

Comment:
Why do these tests now have to change their working dir to appear as if they are in the provider directory?